manage.py

```diff
@@ -4,7 +4,7 @@
 
 if __name__ == "__main__":
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "fund_raiser.settings.local")
-
+    sys.setrecursionlimit(2000)
```

Comment:
@febinstephen : I don't think this is required now, can you please reverify if and remove if unnecessary ?